internal/lookoutingester/instructions/instructions.go

```diff
@@ -1,185 +1,134 @@
 package instructions
 
 import (
-	"context"
 	"sort"
 	"strings"
 	"time"
 
-	commonmetrics "github.com/G-Research/armada/internal/common/ingester/metrics"
+	"github.com/G-Research/armada/internal/common/ingest/metrics"
 
 	"github.com/gogo/protobuf/proto"
 	"github.com/google/uuid"
-	"github.com/grpc-ecosystem/go-grpc-middleware/logging/logrus/ctxlogrus"
 	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
 	log "github.com/sirupsen/logrus"
 	"k8s.io/utils/pointer"
 
+	"github.com/G-Research/armada/internal/common/ingest"
+
 	"github.com/G-Research/armada/internal/common/compress"
 	"github.com/G-Research/armada/internal/common/eventutil"
-	"github.com/G-Research/armada/internal/common/requestid"
 	"github.com/G-Research/armada/internal/common/util"
 	"github.com/G-Research/armada/internal/lookout/repository"
 	"github.com/G-Research/armada/internal/lookoutingester/model"
-	"github.com/G-Research/armada/internal/pulsarutils"
-	"github.com/G-Research/armada/internal/pulsarutils/pulsarrequestid"
 	"github.com/G-Research/armada/pkg/armadaevents"
 )
 
 type HasNodeName interface {
 	GetNodeName() string
 }
 
-type Service struct {
-	m *commonmetrics.Metrics
-}
-
-func New(m *commonmetrics.Metrics) *Service {
-	return &Service{m: m}
+type InstructionCenverter struct {
```

Comment:
```suggestion
type InstructionConverter struct {
```
Should be changed everywhere it's used as well